docker/irods_client/tests/test_data_ops.py

```diff
@@ -51,9 +51,14 @@ def test_upload_download_dataset(session, testdata):
     assert len(ops.upload) == 0
     with ipath.open("w") as handle:
         handle.write("test".encode())
-    ops = upload(session, testdata/"plant.rtf", ipath, overwrite=False, ignore_err=True)
+    ops = upload(session, testdata/"plant.rtf", ipath, overwrite=False, on_err='skip')
```

Comment:
I would prefer `on_error` instead of `on_err` for legibility. It's not too long, right?